src/Common/Integrations/Plugin_Merge_Provider_Abstract.php

```diff
@@ -124,6 +135,20 @@ protected function is_child_plugin_active(): bool {
 		return is_plugin_active( $this->get_plugin_file_key() );
 	}
 
+	/**
+	 * This fires if we are initializing the merge and stores the parent plugin information.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function register_update(): void {
+		// Stores the upgrade string.
+		if( ! in_array( $this->get_plugin_updated_name() , self::$plugin_updated_names ) ) {
```

Comment:
Let's use strict for lookup.
```suggestion
		if( ! in_array( $this->get_plugin_updated_name(), self::$plugin_updated_names, true ) ) {
```